src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/DefaultDisplayTemplates.cs

```diff
@@ -64,24 +64,9 @@ internal static List<SelectListItem> TriStateValues(bool? value)
         {
             return new List<SelectListItem>
             {
-                new SelectListItem
-                {
-                    Text = Resources.Common_TriState_NotSet,
-                    Value = string.Empty,
-                    Selected = !value.HasValue
-                },
-                new SelectListItem
-                {
-                    Text = Resources.Common_TriState_True,
-                    Value = "true",
-                    Selected = (value == true),
-                },
-                new SelectListItem
-                {
-                    Text = Resources.Common_TriState_False,
-                    Value = "false",
-                    Selected = (value == false),
-                },
+                new SelectListItem(Resources.Common_TriState_NotSet, string.Empty, !value.HasValue),
+                new SelectListItem(Resources.Common_TriState_True, "true", (value == true)),
```

Comment:
I think this 3-param constructor doesn't exist anymore.

Reply:
I've made the changes requested, the build works now.